triagebot.toml

```diff
@@ -415,12 +415,19 @@ trigger_files = [
 [autolabel."A-testsuite"]
 trigger_files = [
     "src/ci",
-    "src/tools/compiletest",
     "src/tools/cargotest",
-    "src/tools/tidy",
+    "src/tools/compiletest",
+    "src/tools/miropt-test-tools",
     "src/tools/remote-test-server",
     "src/tools/remote-test-client",
-    "src/tools/tier-check"
+    "src/tools/rustdoc-gui-test",
+    "src/tools/suggest-tests",
+    "src/tools/tier-check",
```

Comment:
I feel like this alone doesn't make sense here. I wonder if replacing this with `src/bootstrap/src/core/build_steps/test.rs` would be better :thinking:

Reply:
Do you mean the `tier-check` one?

Reply:
Yes

Reply:
That makes more sense to me yeah.